src/instance.rs

```diff
@@ -65,6 +65,80 @@ impl<T> Py<T> {
         Ok(ob)
     }
 
+    /// Immutably borrows the value `T`. This borrow lasts untill the returned `PyRef` exists.
```

Comment:
How about just linking `PyCell::borrow`?

Reply:
Agreed I'll simplify all this.